crates/tests/nightly_implement/tests/cast_self.rs

```diff
@@ -5,12 +5,12 @@ use windows::UI::Xaml::*;
 // TODO: This is a compile-only test for now until #81 is further along and can provide composable test classes.
```

Comment:
No, we should create one. I've been reluctant to delve into this as it would take a few hundred lines of code to test and only applies to Xaml which I've been contemplating dropping support for entirely as it is largely unusable in Rust anyway.